### PR TITLE
Adds prefix to cleanup, adds spacing to help visual separation of ID …

### PR DESCRIPTION
**Pull request by ashew**
…and hostname

Since hostnames will often have dashes now with prefixes added, it
makes the whole thing a bit of a mess to read.

### CODE REVIEW

File: lib/qops/deployment/instances.rb [hunk 1]
```diff
@@ -152,7 +152,7 @@ def clean
 
     # Find all instances to be destroyed
     retrieve_instances.each do |instance|
-      next if instance.hostname == 'master'
+      next if instance.hostname == "#{config.hostname_prefix}master"
```
Comment (mbasset) [inline lib/qops/deployment/instances.rb]: perhaps we should use tagging for this.

Comment (mbasset) [inline lib/qops/deployment/instances.rb]: or for now if it regex matches `master`

`next if instance.hostname =~ /master/

Comment (ashew) [inline lib/qops/deployment/instances.rb]: Does the regex really make sense? Is it possible the word "master" might show up in a valid branch name? I think my solution keeps us closer to the intended spirit, which is JUST the master branch, just bringing into account the already-configured prefix value.

I agree with tagging though - definitely more elegant than hostnames.

Comment (mbasset) [inline lib/qops/deployment/instances.rb]: Hopefully with some additional changes we can move to fully using tagging. Should be fine either way.

**Comment by mbasset** [comment]
:+1:

**Pull request merged**